src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2891,20 +2889,26 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             $container->removeDefinition($classToServices[NotifierBridge\Mercure\MercureTransportFactory::class]);
         }
 
-        if (ContainerBuilder::willBeAvailable('symfony/fake-chat-notifier', NotifierBridge\FakeChat\FakeChatTransportFactory::class, ['symfony/framework-bundle', 'symfony/notifier', 'symfony/mailer'])) {
-            $container->getDefinition($classToServices[NotifierBridge\FakeChat\FakeChatTransportFactory::class])
+        // don't use ContainerBuilder::willBeAvailable() as these are not needed in production
+        if (class_exists(NotifierBridge\FakeChat\FakeChatTransportFactory::class)) {
```

Comment:
What about using a `use` statement and simplify here to:

```suggestion
        if (class_exists(FakeChatTransportFactory::class)) {
```